### ISSUE TITLE
Corrupted Weather Data? - Carrol, Iowa

### ISSUE THREAD

**Issue by jessecambon** [NREL/EnergyPlus#6351]
Issue overview
--------------
TLDR: The EnergyPlus (.EPW) weather file for Carrol, Iowa has a maximum temperature of 138 F. This is obviously not correct, but it is actually backed up by weather underground data (in other words they are using the same source). I'm curious to know what organization(s) maintain the weather files and what data quality checks are typically performed.
---------------------------------------------------------

This is an issue related to the weather data available at https://energyplus.net/weather, not to the EnergyPlus program itself so if this is in the wrong place, my apologies.

As a side-project, I made a python script that scrapes and analyzes .EPW EnergyPlus weather files from the above link (https://github.com/cambonator/Weather-Surveillance). To test it, I calculated the 10 locations with the hottest temperature (dry bulb). I was surprised to find Carrol, Iowa on this list (filename 'USA_IA_Carroll.Muni.AP.725468_TMY3.epw').

According to this .EPW file, the city recorded a scorching 138 F (59 C) day on 9/28/1997. Interestingly, weather underground backs up this claim so it must be using the same source (link below). I find this a bit hard to believe since the record temperature on the day prior was 90 F and the hottest recorded temperature on earth was 134 F (death valley July 10 1913). 

Carrol, Iowa has a population of 10,000 people so this issue probably isn't that urgent for most building designers, but I thought it would be worth pointing out as a data integrity issue. There are actually a few other locations that have unbelievably high temperatures in the data. 

I'm curious to know what organization(s) are responsible for maintaining the weather data and what data quality checks are typically performed.

Thanks!

Jesse 

Weather underground page for Carrol, Iowa: https://www.wunderground.com/history/airport/KCIN/1997/9/28/DailyHistory.html?req_city=&req_state=&req_statename=&reqdb.zip=&reqdb.magic=&reqdb.wmo=&MR=1

**Comment by jasondegraw** [comment]
@jessecambon Apparently we didn't have a "weather" label so I didn't see this when you reported it. I've fixed that and I've reported this upstream, so I'm going to close this issue here. Thanks for the report, let us know if you encounter other issues.
